Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -17,34 +17,22 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Lokale 7-Tage-Inzidenz Hospitalisierung</item>
-        <item>Änderung des Links zu Häufige Fragen</item>
-        <item>Zuordnung von Zertifikaten</item>
-        <item>Zertifikate wiederherstellen</item>
+        <item>Papierkorbfunktion für Tests</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
-        <item>Auf den lokalen Kacheln zur 7-Tage-Inzidenz der Neuinfektionen wird nun zusätzlich die 7-Tage-Inzidenz der Hospitalisierung angezeigt. Da diese Kennzahl nur auf Bundeslandebene verfügbar ist, wird bei Auswahl eines Kreises die Hospitalisierung zum jeweiligen Bundesland angezeigt.</item>
-        <item>Wenn Sie in der App die häufigen Fragen aufrufen, werden Sie nun auf Seiten unter coronawarn.app geleitet. Dort finden Sie mehr Details zu Ihren Fragen.</item>
-        <item>Die Zuordnung von Zertifikaten zu Personen wurde fehlertoleranter gestaltet. Die App ignoriert nun eine unterschiedliche Anzahl von Leerzeichen im Namen, sodass die Zertifikate unabhängig davon der gleichen Person zugeordnet werden können.</item>
-        <item>Entfernte Zertifikate werden in einen Papierkorb verschoben. Sie können sie innerhalb von 30 Tagen wiederherstellen.</item>
+        <item>Sie können in der App Schnelltests und PCR-Tests nach dem Löschen wiederherstellen. Sie haben somit die Möglichkeit, Objekte, die sie versehentlich gelöscht haben, wiederherzustellen.</item>
```

Comment:
There's one comma too many - please remove the comma after "Möglichkeit".

Reply:
That's not true. Please check https://www.duden.de/sprachwissen/sprachratgeber/Das-Komma-beim-Infinitiv-mit-%E2%80%9Ezu%E2%80%9C. Because we use Möglichkeit like in the following sentence:
"Vorsicht geboten ist allerdings, wenn solche Verben durch ein auf den Infinitiv verweisendes Wort ergänzt werden, in solchen Fällen ist ein Komma obligatorisch: Wir bitten darum, das Produkt bei Nichtgefallen zurückzusenden. Er glaubt fest an die Möglichkeit, sie mit seinen Argumenten überzeugen zu können."

Reply:
@KathrinLuecke

Reply:
@chiljamgossow But in the sentences as it is now ( Sie haben somit die Möglichkeit, Objekte, die sie versehentlich gelöscht haben, wiederherzustellen.), there is a comma after Möglichkeit and after Objecte and this doesn't look right. The main sentence is "Sie haben somit die Möglichkeit Objekte wiederherzustellen. Then we have a "eingeschobenen Nebensatz" (die Sie versehentlich gelöscht haben) which needs to be separated by commas. So 1 comma after the first part of the main sentence "Sie haben somit die Möglichkeit Objekte, " and the other after the end of the Nebensatz "die Sie versehentlich gelöscht haben".
@CV113 What would you say?

Reply:
the first comm is because of the Infinitivsatz, the second on because of eingeschobener Nebenasatz. You also notice that there needs to be a comma when you speak the sentence out loud, there are small pauses where the commas need to be.